aws/resource_aws_athena_named_query.go

```diff
@@ -14,6 +14,10 @@ func resourceAwsAthenaNamedQuery() *schema.Resource {
 		Read:   resourceAwsAthenaNamedQueryRead,
 		Delete: resourceAwsAthenaNamedQueryDelete,
 
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
 		Schema: map[string]*schema.Schema{
 			"name": &schema.Schema{
```

Comment:
Can we also remove these `&schema.Schema` in this work?
Those are not needed since Go 1.7! 😄